modules/ztp-example-hub-template-functions.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: REFERENCE
+[id="ztp-example-hub-template-functions_{context}"]
+= Example hub templates
+
+The following code examples are valid hub templates. Each of these templates return values from the `ConfigMap` CR with the name `test-config` in the `default` namespace.
+
+.Example hub templates
+[cols=2*, options="header"]
+|====
+|Hub template
+|Description
+
+a|
+[source,yaml]
+----
+{{hub fromConfigMap "default" "test-config" "common-key" hub}}
+----
+|Returns the value with the key `common-key`
+
+a|
+[source,yaml]
+----
+{{hub fromConfigMap "default" "test-config" (printf "%s-name" .ManagedClusterName) hub}}
+----
+|Returns a string by using the `.ManagedClusterName` field that resolves to the name of the target cluster
+
+a|
+[source,yaml]
+----
+{{hub (printf "%s-name" .ManagedClusterName) \| fromConfigMap "default" "test-config" \| toInt hub}}
+----
+|Returns a boolean value based on whether the input parameters exist
+
+a|
+[source,yaml]
+----
+{{hub (printf "%s-name" .ManagedClusterName) \| fromConfigMap "default" "test-config" \| toInt hub}}
+----
+|Casts and returns an integer value based on the input parameters
```

Comment:
Line 34 and 41 are the same but the text says they return different values. Is line 34 wrong?

Reply:
Updated - also added q in slack